yi-frontend-vty/yi-frontend-vty.cabal

```diff
@@ -26,10 +31,16 @@ library
     , pointedlist
     , stm >= 2.2
     , text
-    , vty >= 5.4 && < 6
```

Comment:
If you don't care about supporting vty 5 anymore, then this line can be replaced with:
```
   , vty >= 6
```
And the rest of this file's changes can be reverted, and all the CPP removed.